modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/apirequest/Output.scala

```diff
@@ -117,6 +117,9 @@ final case class JsonOutput(jsonPath: String) extends ApiOutput[Json] {
       case other => other.leftMap(JsonPathException.apply)
     }
 
+  private[this] val jsonFactory: JsonFactory = new JsonFactory(CirceUtils.mapper)
   def parseResponse(response: String): Either[Throwable, Json] =
-    parse(response)
+    Either
+      .catchNonFatal[JsonParser](jsonFactory.createParser(response))
+      .flatMap(parser => Either.catchNonFatal[Json](CirceUtils.mapper.readValue(parser, classOf[Json])))
```

Comment:
```suggestion
    Either
      .catchNonFatal[Json] { 
        val parser = jsonFactory.createParser(response)
        CirceUtils.mapper.readValue(parser, classOf[Json])
      }
```

No reasons to compose these - it's just unnecessary boxing.